src/frontend/scss/_main.scss

```diff
@@ -1,3 +1,6 @@
+// cunningham
+@import './cunningham-tokens';
```

Comment:
Related to another comment: you may need to import `@import "@openfun/cunningham-react/style";` to get all available utility classes. 

Exemple: https://github.com/openfun/cunningham/blob/main/apps/demo/src/index.scss

Reply:
I added a comment on the "next" section of the PR description.
What about keeping this PR about: having cunningham working, simply be able to use cunningham components.
Then we list what's we wanna use and implement it piece by piece ?